vncdotool/api.py

```diff
@@ -81,8 +81,14 @@ def __init__(self, factory):
         self.factory = factory
         self.queue = queue.Queue()
 
-    def connect(self, host, port=5900):
-        reactor.callWhenRunning(reactor.connectTCP, host, port, self.factory)
+    def connect(self, server):
```

Comment:
This is a backwards incompatible change to the API.  I'm not sure if the right answer is a new method, new kwarg or something else.